### PR TITLE
Fw 1677

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

File: packages/core/test/acceptance/view_insertion_spec.ts [hunk 1]
```diff
@@ -541,4 +541,74 @@ describe('view insertion', () => {
...
+      const fixture = TestBed.createComponent(AppComponent);
+      fixture.detectChanges();
+      expect(fixture.nativeElement.textContent).toBe('container start||container end|click');
+
+      fixture.componentInstance.click();
+      fixture.detectChanges();
+      expect(fixture.nativeElement.textContent).toBe('container start|test|container end|click');
+    });
+
+    it('should properly insert before views in a ViewContainerRRef injected on ng-container',
```
Comment (mhevery) [inline packages/core/test/acceptance/view_insertion_spec.ts]:
```suggestion
    it('should properly insert before views in a ViewContainerRef injected on ng-container',
```

File: packages/core/test/acceptance/view_insertion_spec.ts [hunk 2]
```diff
@@ -541,4 +541,74 @@ describe('view insertion', () => {
...
+         }
+
+         TestBed.configureTestingModule({
+           declarations: [AppComponent],
+           imports: [CommonModule],
+         });
+
+         const fixture = TestBed.createComponent(AppComponent);
+         fixture.detectChanges();
+
```
Comment (mhevery) [inline packages/core/test/acceptance/view_insertion_spec.ts]: could you add intermediary assertion?
```suggestion

expect(fixture.nativeElement.textContent.trim()).toContain('1');
```

**Review by mhevery — APPROVED** [review]
Not sure this is ready for review but LGTM

File: packages/core/test/acceptance/view_insertion_spec.ts (hunk 2 above)
Comment (pkozlowski-opensource) [reply packages/core/test/acceptance/view_insertion_spec.ts]: Done

File: packages/core/test/acceptance/view_insertion_spec.ts (hunk 1 above)
Comment (pkozlowski-opensource) [reply packages/core/test/acceptance/view_insertion_spec.ts]: Done

**Comment by mhevery** [comment]
[presubmit](http://test/OCL:281089486:BASE:281089528:1574099266006:944d25ce)
[ivy presubmit](http://test/OCL:281089486:BASE:281089748:1574099320708:27b2e2d)

**Comment by pkozlowski-opensource** [comment]
merge-assistance since:
* "Status "google3" is pending"
* approval status shows that `fw-core` needs to approve while Misko approved...

**Review by AndrewKushnir — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
@pkozlowski-opensource VE and Ivy presubmits are successful, I've updated g3 status on this PR. Thank you.

**Pull request closed without merging**